ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/MissionItemType.java

```diff
@@ -113,6 +114,18 @@ public MissionItem getNewItem() {
         }
     },
 
+    DO_LAND_START("Do Land start") {
```

Comment:
@daimonkor I have the base `isTypeSupported(Type vehicleType)` method implemented [here](https://github.com/DroidPlanner/DroneKit-Android/blob/api_bug_fixes/ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/MissionItemType.java#L263-L270) for reference.

For completion, the `DO_LAND_START` should override the base method, and only return true when the vehicle is of the plane type.

Consumer of this method would the client apps connecting to 3DR Services. For example in Tower, currently the supported mission items are hardcoded as shown [here](https://github.com/DroidPlanner/Tower/blob/91e3c5adbc524c7d4702f3a934487486edb91957/Android/src/org/droidplanner/android/proxy/mission/item/fragments/MissionDetailFragment.java#L53-L67). That rigid structure can be replaced by enumerating through the list of `MissionItemType` objects and including those for which `isTypeSupported(...)` returns true.